docs/receipt-verifier.md

```diff
@@ -0,0 +1,98 @@
+---
+id: receipt-verifier
+title: Receipt Verifier Service
+sidebar_label: Receipt verifier service
+---
+
+This page explains how to set up a receipt verifier service.
+
+As more creators adopt Web Monetization to serve premium and/or ad-free content, there’s a greater need for reliable payment verification. Web Monetization browser events, like `monetizationstart`, are helpful indicators of payment, but can be spoofed by savvy users to access exclusive content without paying.
+
+[Interledger STREAM receipts](https://interledger.org/rfcs/0039-stream-receipts/) provide payment recipients (such as web monetized site owners) with verifiable proofs of payment. Payment recipients can verify STREAM receipts themselves by setting up their own verification service or by using a third-party service.
+
+## Before you begin
+
+Your Web Monetization receiver (the digital entity receiving payments) must be set up to generate Interledger STREAM receipts. GateHub and Uphold already support receipts.
+
+## Service overview
+
+The receipt verifier service acts as a proxy between the Web Monetization sender and receiver.
+
+Before a payment is made, the WM sender sends a query to the WM receiver. The query passes through the verifier service, which adds a receipt secret and receipt nonce.
+
+When the WM receiver receives money from the payment, it generates a receipt (which includes a signature generated using the receipt secret) and sends the receipt to the WM sender.
+
+The WM sender passes the receipt to the verifier service so the verifier can confirm the payment (as only the WM receiver and verifier service know the receipt secret). The verifier must verify the receipt before accepting the amount as paid.
+
+## Install the receipt verifier service package
+
+You can get the [receipt verifier service package](https://github.com/coilhq/receipt-verifier) or run the verifier as a [Kubernetes service](https://github.com/coilhq/receipt-verifier/tree/main/config/base).
+
+**Example**
+
+We use Redis in this example. You can make a verifier that doesn't use Redis.
+
+```
+npm install
+npm run-script build
+sudo docker run -p 6379:6379 -d redis
+npm start
+```
+
+## Update your Web Monetization meta tag
+
+Typically, a Web Monetization meta tag looks something like this:
+
+`<meta name=“monetization" content="$wallet.example.com/alice">`
+
+Update your meta tag so that your URL-encoded payment pointer is in the path of the receipt verifier service’s URL.
```

Comment:
We should link somewhere like https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent to give people a little help on URI-encoding their payment pointer